driver/razerchromacommon.c

```diff
@@ -539,8 +539,8 @@ struct razer_report razer_chroma_extended_matrix_effect_static(unsigned char var
  * Seems like direction is now 0x00, 0x01 for Left/Right, used to be 0x01, 0x02
  *
  * Status Trans Packet Proto DataSize Class CMD Args
- * 00     3f    0000   00    06       0f    02  010504002800 | SET LED MATRIX Effect (VARSTR, Backlight, Wave 0x04, Dir 0x00, ? 0x2800)
- * 00     3f    0000   00    06       0f    02  010504012800 | SET LED MATRIX Effect (VARSTR, Backlight, Wave 0x04, Dir 0x01, ? 0x2800)
+ * 00     3f    0000   00    06       0f    02  010504002800 | SET LED MATRIX Effect (VARSTR, Backlight, Wave 0x04, Dir 0x00, ? 0x2100)
+ * 00     3f    0000   00    06       0f    02  010504012800 | SET LED MATRIX Effect (VARSTR, Backlight, Wave 0x04, Dir 0x01, ? 0x2100)
```

Comment:
heh, if you edit it here then also edit the hex version (e.g. 010504012800 => 010504012100)

Reply:
D'Oh! :man_facepalming: